### DISCUSSION TITLE
Option to disable highlights for files and folders separately

### DISCUSSION THREAD

**Discussion by Machine-Maker** [nvim-tree/nvim-tree.lua#2196] [Ideas]
Afaik, there isn't a setting to differentiate the `git_highlight` between files and folders. It'd be nice to be able to configure those separately. In fact, I think quite a few of the settings would benefit from having the ability to configure files and folders separately.

**Comment by alex-courtis** [comment]
That is possible, perhaps `NvimTreeFolderDirty`, `NvimTreeFolderStaged` etc.

Simplest to link those both to `NvimTreeGitDirty` etc.

**Comment by Machine-Maker** [comment]
Are you saying its possible now? or possible to add those highlight groups? I don't see those documented anywhere if they exist.

**Reply by alex-courtis** [reply]
Created #2210 

As usual, pull requests are gratefully appreciated. See [Development](https://github.com/nvim-tree/nvim-tree.lua/wiki/Development)
